shared/src/main/scala/za/co/absa/hyperdrive/shared/utils/ConfigUtils.scala

```diff
@@ -41,4 +41,18 @@ object ConfigUtils {
       case Failure(exception) => None
     }
   }
+
+  def getPropertySubset(configuration: Configuration, prefix: String): Option[Map[String, String]] = {
+    val subset = configuration.subset(prefix)
+    if (!subset.isEmpty) {
+      import scala.collection.JavaConverters._
+      val keys = subset.getKeys()
+      val properties = keys.asScala
+        .map(key => (key, getOrThrow(key, subset)))
```

Comment:
getOrThrow is no best name.
It is not your method so you can leave it.

Reply:
+ in method getOrThrow is errorMessage: String = "".
Better would be to have an error message with a generic message ("property.1.2.4" doesn't exist or so) in case of an empty string.

Reply:
Created #76